ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+import six
+import pytest
+
+import pandas as pd
+import ibis.util
+import ibis.bigquery
```

Comment:
Just `import ibis`. `ibis.bigquery` may not exist if someone doesn't have `google-cloud-bigquery` installed. This module should be importable without the required dependencies so that pytest can still run.